eng/testing/mobile/outerBuild.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Target Name="Test" DependsOnTargets="GetProjectWithBestTargetFrameworks">    
+    <MSBuild Projects="@(InnerBuildProjectsWithBestTargetFramework)"
```

Comment:
Build in parallel?

Reply:
Invoking that target in parallel is inconvenient as it means that test execution for multiple target frameworks runs in parallel and the output interleaves.

Would it make sense to set BuildInParallel to true if all test projects are being tested (ie from root) and set it to false when an individual project is being tested? (the default sdk behavior is that when an individual project is being tested, the test invocations are serial).

Reply:
Someone else will have to answer that - I'm just used to reflexively adding it everywhere, excepting where there's a specific reason not to (as perhaps here). It's a shame that for back compat, it's still necessary to add it explicitly.

Reply:
I changed the code to only build in parallel when tested from root. Thanks.